keyboards/crkbd/keymaps/default/rules.mk

```diff
@@ -1,2 +1,5 @@
-OLED_ENABLE = yes
-OLED_DRIVER = SSD1306
+MOUSEKEY_ENABLE = no     # Mouse keys
```

Comment:
What's the reasoning behind disabling mouse key support? Is it just that the default keymap doesn't use them?

My concern here being the new-keymap command copies these rules and the user might be used to mouse keys working from configurator. Although admittedly it is a easy fix but it's still a often asked question.

Reply:
Thanks for pointing it out. It was an error on my part from copying the `rules.mk` over from `via` keymap. I've reduced the number of effects for the `via` keymap as well to fit mouse key.